packages/components/src/popover/index.js

```diff
@@ -388,15 +401,14 @@ const Popover = (
 		}
 
 		const { defaultView } = ownerDocument;
-		const { frameElement } = defaultView;
 
 		ownerDocument.addEventListener( 'scroll', update );
 
 		let updateFrameOffset;
-		if ( frameElement ) {
+		if ( defaultView ) {
```

Comment:
Is there a case where `defaultView` is not defined? 🤔

Reply:
[It can be `null` according to MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/defaultView), but it doesn't say under what circumstances.

Reply:
Okay, it's probably about the running environment. Why not check for `frameElement` directly before adding the listener?

Reply:
Sure, have pushed a PR that does that. 👍 

Also moved the `getFrameOffset` function to the utils file. One of Marco's PRs that was just merged moved all the other functions so after rebasing this one was a bit lonely 😄

Thanks for the reviews!